cmd/server.go

```diff
@@ -46,7 +47,15 @@ var (
 	}
 )
 
+func patchTiDBConf() {
+	conf := config.NewConfig()
```

Comment:
we can update the tidb config like this:
https://github.com/pingcap/ticdc/pull/1115/files#diff-fc79b7274e95fe44af88567f9dbe6e2c49c386a923698cc7541157b5747ff079R870-R872

Reply:
addressed